### DISCUSSION TITLE
[Feature] HarmonyOS NEXT app?

### DISCUSSION THREAD

**Discussion by xu75** [immich-app/immich#14974] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have both iPhone and Huawei Mobile Phone, since Huawei is changing to HarmonyOS NEXT these days, my photos from huawei can not be upload to NAS automatically any more. (I was using Synology Photos Android APP to backup Huawei Photos to my Synology NAS server, and iCloudPhotoDownloader for icloud)

Could you guys to support HarmonyOS NEXT mobile app in order to synchronise with Huawei Mobile Phone? if YES, i will change immediately from Synology Photos to IMMICH.

Anyway, thank you so much for the great IMMICH

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#13571
